src/screens/EventSuggestionScreen.tsx

```diff
@@ -0,0 +1,250 @@
+
+import {
+  Button,
+  Checkbox,
+  DefaultKeyboardAvoidingView,
+  HeadlineText,
+  HtmlView,
+  Input,
+  RegularText,
+  SafeAreaViewFlex,
+  Wrapper,
+  WrapperVertical
+} from '../components';
+import { colors, consts, normalize, texts } from '../config';
+import { useMatomoTrackScreenView } from '../hooks';
+import { createQuery, QUERY_TYPES } from '../queries';
+
+const { MATOMO_TRACKING, EMAIL_REGEX } = consts;
+
+type TAppUserContent = {
+  address: string;
+  consent: boolean;
+  email: string;
+  eventName: string;
+  message: string;
+  name: string;
+  phone: string;
+};
+
+export const EventSuggestionScreen = ({
+  navigation,
+  route
+}: {
+  navigation: NavigationProp<any>;
+  route: { params: { formIntroText: string } };
+}) => {
+  const [loading, setLoading] = useState(false);
+  const introText = route.params.formIntroText || '';
+
+  const {
+    control,
+    formState: { errors },
+    reset,
+    handleSubmit
+  } = useForm({
+    defaultValues: {
+      address: '',
+      consent: false,
+      email: '',
+      eventName: '',
+      description: '',
+      name: '',
+      phone: ''
+    }
+  });
+
+  useMatomoTrackScreenView(MATOMO_TRACKING.SCREEN_VIEW.FEEDBACK);
+
+  const [createAppUserContent] = useMutation(createQuery(QUERY_TYPES.APP_USER_CONTENT));
+
+  const onSubmit = async (createAppUserContentNewData: TAppUserContent) => {
+    Keyboard.dismiss();
+
+    if (!createAppUserContentNewData.consent) {
+      return Alert.alert(
+        texts.feedbackScreen.inputsErrorMessages.hint,
+        texts.feedbackScreen.inputsErrorMessages.checkbox
+      );
+    }
+
+    const formData = {
+      dataType: 'json',
+      dataSource: 'form_event_suggestion',
+      content: JSON.stringify({
+        action: 'event_suggestion',
+        address: createAppUserContentNewData.address,
+        consent: createAppUserContentNewData.consent,
+        email: createAppUserContentNewData.email,
+        eventName: createAppUserContentNewData.eventName,
+        description: createAppUserContentNewData.description,
+        name: createAppUserContentNewData.name,
+        phone: createAppUserContentNewData.phone
+      })
+    };
+
+    setLoading(true);
+
+    try {
+      await createAppUserContent({ variables: formData });
+
+      reset();
+
+      Alert.alert(
+        texts.eventSuggestionScreen.alert.title,
+        texts.eventSuggestionScreen.alert.message
+      );
+      navigation.goBack();
+    } catch (error) {
+      console.error(error);
+    }
+    setLoading(false);
+  };
+
+  return (
+    <SafeAreaViewFlex>
+      <DefaultKeyboardAvoidingView>
+        <ScrollView keyboardShouldPersistTaps="handled">
+          <Wrapper>
+            <HtmlView html={introText} />
+
+            <WrapperVertical>
+              <Divider />
+            </WrapperVertical>
+            <HeadlineText big>{texts.eventSuggestionScreen.eventOwnerInfo}</HeadlineText>
+            <Input
+              boldLabel
```

Comment:
let's not habe divider in the form. the big headlines are already dividing enough. i have changed this to:

<img src="https://github.com/user-attachments/assets/049e6e21-0d94-41de-8de8-7e8cc7324fc2" width="250" />